datasets/wikihow/wikihow.py

```diff
@@ -107,6 +96,19 @@ class Wikihow(nlp.GeneratorBasedBuilder):
         WikihowConfig(name="sep", filename="wikihowSep.csv", description="use each paragraph and its summary."),
     ]
 
+    @property
+    def manual_download_instructions(self):
+        return """\
+  You need to manually download two wikihow files. An overview of which files to download can be seen at https://github.com/mahnazkoupaee/WikiHow-Dataset.
+  You need to download the following two files manually:
+    1) https://ucsb.app.box.com/s/ap23l8gafpezf4tq3wapr6u8241zz358 and save the file under <path/to/folder>/wikihowAll.csv
+    2) https://ucsb.app.box.com/s/7yq601ijl1lzvlfu4rjdbbxforzd2oag and save the file under <path/to/folder>/wikihowSep.csv
+
+  The <path/to/folder> can e.g. be "~/manual_wikihow_data".
+
+  Wikihow can then be loaded using the following command `nlp.load("wikihow", data_file="<path/to/folder>")`.
```

Comment:
```suggestion
  Wikihow can then be loaded using the following command `nlp.load("wikihow", data_dir="<path/to/folder>")`.
```
It should be `data_dir` ?